src/react/projects/spark-react/src/SprkInput/SprkTextInput/components/SprkLabelLocationCheck.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import classNames from 'classnames';
+import propTypes from 'prop-types';
+
+const SprkLabelLocationCheck = ({
+  children,
+  type,
+  label,
+  leadingIcon,
+  hiddenLabel,
+  id,
+}) => {
+  if (type === 'hugeTextInput') {
+    return (
+      <>
+        {children}
+        <label
```

Comment:
Hey @RVMendoza, we should make the label a presentational component that this SprkLabelLocationCheck uses. We can pull out the label code and make a new SprkLabel.js file.